testnets/quasartestnet/chain.json

```diff
@@ -32,21 +32,35 @@
   },
   "codebase": {
     "git_repo": "https://github.com/quasar-finance/quasar",
-    "recommended_version": "v1.0.1-rc-testnet",
+    "recommended_version": "v2.0.0-rc",
     "compatible_versions": [
-      "v1.0.1-rc-testnet"
+      "v2.0.0-rc"
     ],
-    "cosmos_sdk_version": "0.45.16",
+    "cosmos_sdk_version": "0.47.12",
     "consensus": {
-      "type": "tendermint",
-      "version": "0.34.27"
+      "type": "cometbft",
+      "version": "0.37.6"
     },
-    "cosmwasm_version": "0.31",
+    "cosmwasm_version": "0.45.0",
     "cosmwasm_enabled": true,
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/quasar-finance/questnet/main/v04/definitive-genesis.json"
     },
     "versions": [
+      {
+        "name": "v2.0.0-rc",
```

Comment:
Please relocate the latest version to the bottom of the versions array

Reply:
Sure.